addons/packages/whereabouts/0.5.0/README.md

```diff
@@ -0,0 +1,108 @@
+# whereabouts Package
+
+This package provides the ability to assign IP addresses dynamically across your Kubernetes cluster using a CNI IPAM plugin named [whereabouts](https://github.com/k8snetworkplumbingwg/whereabouts).
+
+## Components
+
+* Whereabouts Custom Resources
+* Whereabouts DaemonSet
+* Whereabouts ServiceAccount
+* Whereabouts ClusterRoleBinding
+* Whereabouts ip-reconciler Cronjob
+
+## Configuration
+
+The following configuration values can be set to customize the whereabouts installation.
+
+### Global
+
+| Value | Required/Optional | Description |
+|-------|-------------------|-------------|
+| `namespace` | Optional | The namespace in which to deploy whereabouts components. Default: kube-system |
+
+### whereabouts Configuration
+
+| Value | Required/Optional | Description |
+|-------|-------------------|-------------|
+| `whereabouts.config.resources.limits.cpu` | Optional | The limits for CPU resources of whereabouts DeamonSet  |
```

Comment:
please make sure if they are optional when providing a values file
For example, if there is no `whereabouts.config.resources.requests.memory`, what is the final content of the affected resources spec.

Reply:
My understanding is that if we are providing something in the `values.yaml`, that means we have set default values, so I set it to `Optional`.

Leave this comment here and see if there're other opinions.

Reply:
Most importantly is that all configuration parameters are documented.